vendor/assets/stylesheets/rolodex/components/_grid-responsive.sass

```diff
@@ -0,0 +1,28 @@
+// Responsive Grid
+// ========================================
+
+// Responsive grid classes from $breakpoints
```

Comment:
this is an optional grid, grid-responsive, not included by default. the basic grid is included by default. I just moved it from the `utilities` folder to `components/grid`